src/nrc/tests/commands/test_setup_configuration.py

```diff
@@ -1,36 +1,25 @@
 import uuid
 from io import StringIO
+from pathlib import Path
 
 from django.contrib.sites.models import Site
-from django.core.management import CommandError, call_command
-from django.test import TestCase, override_settings
+from django.core.management import call_command
+from django.test import TestCase
 from django.urls import reverse
 
-import requests
 import requests_mock
 from jwt import decode
 from rest_framework import status
 from vng_api_common.authorizations.models import AuthorizationsConfig
 from vng_api_common.authorizations.utils import generate_jwt
+from vng_api_common.contrib.setup_configuration.steps import JWTSecretsConfigurationStep
+from zgw_consumers.contrib.setup_configuration.steps import ServiceConfigurationStep
+
+from nrc.setup_configuration.authorization import AuthorizationStep
+
+CONFIG_FILE_PATH = Path("src/nrc/tests/commands/files/setup_config_full.yaml").resolve()
+
 
-from nrc.config.authorization import AuthorizationStep, OpenZaakAuthStep
-from nrc.config.notification_retry import NotificationRetryConfigurationStep
-from nrc.config.site import SiteConfigurationStep
-
-
-@override_settings(
-    SITES_CONFIG_ENABLE=True,
-    OPENNOTIFICATIES_DOMAIN="open-notificaties.example.com",
-    OPENNOTIFICATIES_ORGANIZATION="ACME",
-    AUTHORIZATION_CONFIG_ENABLE=True,
-    AUTORISATIES_API_ROOT="https://oz.example.com/autorisaties/api/v1/",
-    NOTIF_OPENZAAK_CLIENT_ID="notif-client-id",
-    NOTIF_OPENZAAK_SECRET="notif-secret",
-    OPENZAAK_NOTIF_CONFIG_ENABLE=True,
-    OPENZAAK_NOTIF_CLIENT_ID="oz-client-id",
-    OPENZAAK_NOTIF_SECRET="oz-secret",
-    NOTIFICATION_RETRY_CONFIG_ENABLE=True,
-)
 class SetupConfigurationTests(TestCase):
```

Comment:
Not 100% if we need these full tests tbh

Reply:
If the steps are not dependent on other steps from within the project itself I don't see any added value of the file